debian/control

```diff
@@ -17,6 +17,7 @@ Build-Depends: bison,
                libjson-c-dev | libjson0-dev,
                libpam0g-dev | libpam-dev,
                libpcre2-dev,
+               libpcre2-posix3,
```

Comment:
This is not correct and not needed, only the line above is needed. (Just remove it)

Reply:
It was initially (https://github.com/FRRouting/frr/commit/96301f793bd29b8f078c4142e964e4a83835a16e), but failed, just added with the latest version to see what's going on.